src/lib.rs

```diff
@@ -75,6 +75,13 @@ pub use ttf_parser::Width as Stretch;
 use slotmap::SlotMap;
 use tinyvec::TinyVec;
 
+/// A unix file identifier. This is the device ID and the inode number.
+#[cfg(all(unix, feature = "fs"))]
+type FileIdentifier = (u64, u64);
```

Comment:
Can't we use file descriptor here?

Reply:
A fd represents an open file. The options are either the current approach, or a PathBuf. PathBuf would be xplat, but is about 5% slower (again, should only be a factor when a symlink is encountered). Should I switch to that instead?